src/EditorFeatures/VisualBasicTest/SplitComment/SplitCommentCommandHandlerTests.vb

```diff
@@ -0,0 +1,206 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.SplitComment
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.SplitComment
+    <UseExportProvider>
+    Public Class SplitCommentCommandHandlerTests
+        Inherits AbstractSplitCommentCommandHandlerTests
+
+        Protected Overrides Function CreateWorkspace(markup As String) As TestWorkspace
+            Return TestWorkspace.CreateVisualBasic(markup)
+        End Function
+
+        <WorkItem(38516, "https://github.com/dotnet/roslyn/issues/38516")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.SplitComment)>
+        Public Sub TestSplitStartOfComment()
+            TestHandled(
+"Module Program
+    Sub Main(args As String())
+        '[||]Test Comment
+    End Sub
+End Module
+",
+"Module Program
+    Sub Main(args As String())
+        '
+        'Test Comment
+    End Sub
+End Module
+")
+        End Sub
+
+        <WorkItem(38516, "https://github.com/dotnet/roslyn/issues/38516")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.SplitComment)>
+        Public Sub TestSplitStartOfCommentWithLeadingSpace1()
+            TestHandled(
+"Module Program
+    Sub Main(args As String())
+        ' [||]Test Comment
+    End Sub
+End Module
+",
+"Module Program
+    Sub Main(args As String())
+        '
+        ' Test Comment
+    End Sub
+End Module
+")
+        End Sub
+
+        <WorkItem(38516, "https://github.com/dotnet/roslyn/issues/38516")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.SplitComment)>
+        Public Sub TestSplitStartOfCommentWithLeadingSpace2()
+            TestHandled(
+"Module Program
+    Sub Main(args As String())
+        '[||] Test Comment
+    End Sub
+End Module
+",
+"Module Program
+    Sub Main(args As String())
+        '
+        ' Test Comment
+    End Sub
+End Module
+")
+        End Sub
+
+        <WorkItem(38516, "https://github.com/dotnet/roslyn/issues/38516")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.SplitComment)>
+        Public Sub TestSplitMiddleOfComment()
```

Comment:
💡 I didn't see tests for variable leading spaces within the comment. Does VB automatically remove all but the first?

Reply:
tests added.  VB working the same as C# :)